jenkins-scripts/dsl/brew_release.dsl

```diff
@@ -153,36 +150,17 @@ bottle_job_builder.with
 
    touchStoneFilter("label == ${touchstone_label}")
 
-   parameters
-   {
-     stringParam("PULL_REQUEST_URL", '',
-                 'Pull request URL (osrf/homebrew-simulation) pointing to a pull request.')
-     stringParam("BREW_REPO", 'https://github.com/scpeters/brew.git',
-                 'Repository for fork of homebrew/brew.')
-     stringParam("BREW_BRANCH", 'still_working',
-                 'Branch of brew repository to use.')
-     stringParam("TEST_BOT_REPO", 'https://github.com/scpeters/homebrew-test-bot.git',
-                 'Repository for fork of homebrew/homebrew-test-bot.')
-     stringParam("TEST_BOT_BRANCH", 'still_working',
-                 'Branch of homebrew-test-bot repository to use.')
-   }
-
    steps {
         systemGroovyCommand("""\
           build.setDescription(
-          '<b>' + build.buildVariableResolver.resolve('PACKAGE_ALIAS') + '-' +
-           build.buildVariableResolver.resolve('VERSION') + '</b>' +
-          '<br />' +
-          'pull request:<b> <a href="' + build.buildVariableResolver.resolve('PULL_REQUEST_URL') +
-          '">' + build.buildVariableResolver.resolve('PULL_REQUEST_URL') + '</a></b>' +
-          '<br />' +
-          'RTOOLS_BRANCH: ' + build.buildVariableResolver.resolve('RTOOLS_BRANCH'));
```

Comment:
could we leave the RTOOLS_BRANCH line in the description? I find it useful to determine which jobs are under testing or development.

Reply:
I removed it because there isn't a good way to specify that parameter for builds triggered by the ghprb-plugin; it's always the default. The only way I could figure out how to test this branch was to change the default value of `RTOOLS_BRANCH` in d828118b89dda5088f95b503926e5e2e81c99e03.

It would be cool if we were able to post a comment like `RTOOLS_BRANCH=custom_branch build bottle` and parse variables from the `ghprbCommentBody` parameter and change the release-tools branch accordingly. If we did that variable from the comment body, would we be able to set it in the build description?

Reply:
that would be possible I think, [exactly here ](https://github.com/ignition-tooling/release-tools/blob/master/jenkins-scripts/dsl/_configs_/OSRFUNIXBase.groovy#L37-L39). Feel free to open an issue with it, I think it is a good idea.

If you need to modify the value at some point, another option is to change it manually using the /configure URL (or the configure button in the web UI). 

Given that we have these options, maybe a good idea to leave this line in place to report RTOOLS_BRANCH value.

Reply:
ok, I've restored it in aaba7b2c3a9854c9cb98370d2ec7d66946f1d3c3